app/src/cli/mod.rs

```diff
@@ -70,6 +70,19 @@ pub enum Commands {
         /// The server to connect to; defaults to localhost
         host: Option<String>,
     },
+    /// Asset manipulation and migration
```

Comment:
Hm we should strive for making the ambient code base as small as possible. Can you put the migrate stuff in a separate repository? Or just remove it, I don't think anyone has a complex pipeline yet so people can just migrate by hand

Reply:
Mithun mentioned that it'd be best to have it in ambient since others may need to use it when developing.

So his conclusion was to have it as part of our tool

Reply:
I think we should keep it in here for now and then delete it for the next release

Reply:
They can use it even if it's in a separate repo. We need to make sure the codebase doesn't balloon in size with temporary code patches like this. Each line of code is adding build times, it's more code to maintain, it makes it harder to understand (migrate will only be relevant for a few weeks now, then never again).

As a general principle, we should try to minimize the number of lines of code in the codebase. Each line of code should be seen as a cost, so it needs to have a lot of long term benefit to warrant a place in the code base; otherwise it shouldn't be written or it should live in an outside repo.

cc @philpax

Reply:
> I think we should keep it in here for now and then delete it for the next release

Sure that's fine, but @ten3roberts you're on the hook for making sure that happens. Set a calendar reminder for yourself to remove it